src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Amazon Lex monitoring integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's AWS Lex monitoring integration: what data it reports, and how to enable it."
+redirects:
+  - /docs/integrations/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration
+  - /docs/aws-lex-integration
+  - /docs/infrastructure/infrastructure-integrations/amazon-integrations/aws-lex-integration
+  - /docs/infrastructure/amazon-integrations/amazon-integrations/aws-lex-monitoring-integration
+---
+
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an Amazon Lex integration for reporting your Lex data to New Relic. Here we explain how to activate our integration and what data can be reported.
+
+Enable the [AWS CloudWatch Metric Streams integration](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-metric-stream/) to monitor all CloudWatch metrics from your AWS services, including custom namespaces. Individual integrations are no longer our recommended option.
+
+## Features [#features]
+
+[Amazon Lex](https://aws.amazon.com/lex/) (Lex) is an AWS service for building conversational interfaces into applications using voice and text. With New Relic's Lex integration, you'll be able to monitor number of requests, channels used for Bot conversation and more. You'll be able to create custom charts of your data and [create alert conditions](/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information) based on changes in Lex data or configuration.
+
+## Activate integration [#activate]
+
+To enable this integration follow standard procedures to [connect AWS services to New Relic](/docs/infrastructure/infrastructure-integrations/getting-started/connect-aws-integrations-infrastructure).
+
+## Configuration and polling [#polling]
```

Comment:
Remove this section as it is not relevant

Reply:
done